src/main/java/mate/academy/spring/service/Email.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.service;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+
+@Target({FIELD})

```

Comment:
Only field? Or we can use it for parameters?

Reply:
added